src/Elmish.WPF/Binding.fs

```diff
@@ -277,6 +286,63 @@ module Binding =
       vopt
       >> mapModel ValueOption.ofOption
       >> mapMsg ValueOption.toOption
+      
+  module SubModelWin =
+    open BindingData.SubModelWin
+
+    let internal create getState createViewModel updateViewModel toMsg getWindow isModal onCloseRequested =
+      { GetState = getState
+        CreateViewModel = createViewModel
+        UpdateViewModel = updateViewModel
+        ToMsg = toMsg
+        GetWindow = getWindow
+        IsModal = isModal
+        OnCloseRequested = onCloseRequested }
+      |> box
+      |> SubModelWinData
+      |> BaseBindingData
+      |> createBinding
+
+  module SubModelSeqUnkeyed =
+    open BindingData.SubModelSeqUnkeyed
+
+    let internal create createViewModel updateViewModel =
+      { GetModels = id
+        CreateViewModel = createViewModel
+        CreateCollection = ObservableCollection >> CollectionTarget.create
+        UpdateViewModel = updateViewModel
+        ToMsg = fun _ -> id }
+      |> box
+      |> SubModelSeqUnkeyedData
+      |> BaseBindingData
+      |> createBinding
+
+  module SubModelSeqKeyed =
+    open BindingData.SubModelSeqKeyed
+
+    let internal create createViewModel updateViewModel getUnderlyingModel getId =
+      { GetSubModels = id
+        CreateViewModel = createViewModel
+        CreateCollection = ObservableCollection >> CollectionTarget.create
+        UpdateViewModel = updateViewModel
+        GetUnderlyingModel = getUnderlyingModel
+        ToMsg = fun _ -> id
+        GetId = getId }
+      |> box
+      |> SubModelSeqKeyedData
+      |> BaseBindingData
+      |> createBinding
+
+module Bindings =
+
+  /// Maps the model of a list of bindings via a contravariant mapping.
+  let mapModel (f: 'a -> 'b) (bindings: Binding<'b, 'msg> list) = f |> Binding.mapModel |> List.map <| bindings
+
+  /// Maps the message of a list of bindings with access to the model via a covariant mapping.
+  let mapMsgWithModel (f: 'a -> 'model -> 'b) (bindings: Binding<'model, 'a> list) = f |> Binding.mapMsgWithModel |> List.map <| bindings
```

Comment:
```suggestion
  let mapMsgWithModel (f: 'a -> 'model -> 'b) (bindings: Binding<'model, 'a> list) =
    f |> Binding.mapMsgWithModel |> List.map <| bindings
```

Reply:
I'd prefer not to make formatting adjustments at this time

Reply:
I'd prefer not to make formatting adjustments at this time